src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -573,6 +607,11 @@ else if (serial)
             throw new RuntimeException("No StatementDialects is available for " + bench);
         }
 
+        if (argsLine.hasOption("initial-delay")) {
+            int initialDelay = Integer.parseInt(argsLine.getOptionValue("initial-delay"));
+            LOG.info("Sleeping for " + initialDelay);
```

Comment:
Lets change message to:

"Delaying execution of workload for ... seconds"

Reply:
Done